src/org/labkey/test/tests/DataClassTest.java

```diff
@@ -128,6 +131,38 @@ public void testReservedFieldNames()
         createPage.clickCancel();
     }
 
+    @Test
+    public void testIgnoreReservedFieldNames()
+    {
+        final String expectedInfoMsg = "Reserved fields found in your file are not shown below. " +
+                "These fields are already used by LabKey to support this data class: " +
+                "Name, Created, createdBy, Modified, modifiedBy, container, created, createdby, modified, modifiedBy, Container.";
+
+        List<String> lines = new ArrayList<>();
+        lines.add("Name,TextField1,DecField1,DateField1,Created,createdBy,Modified,modifiedBy,container,created,createdby,modified,modifiedBy,Container,SampleID");
+
+        if (!TestFileUtils.getTestTempDir().exists())
+            TestFileUtils.getTestTempDir().mkdirs();
+        File inferenceFile = TestFileUtils.saveFile(TestFileUtils.getTestTempDir(), "InferFieldsForDataClass.csv", String.join(System.lineSeparator(), lines));
+
+        goToProjectHome();
+
+        String name = "Ignore Reserved Fields";
+        log("Create a Data class Type.");
+        CreateDataClassPage createPage = goToCreateNewDataClass();
+        createPage.setName(name);
+
+        log("Infer fields from a file that contains some reserved fields.");
+        DomainFormPanel domainForm = createPage
+                .getDomainEditor()
+                .setInferFieldFile(inferenceFile);
+        checker().verifyEquals("Reserved field warning not as expected",  expectedInfoMsg, domainForm.getPanelAlertText());
+        createPage.clickSave();
+        DataRegionTable drt = DataRegion(getDriver()).find();
+        checker().verifyTrue("Data class not found in list of data classes", drt.getColumnDataAsText("Name").contains(name));
+        log("End of test.");
```

Comment:
I get dinged in my PR feedback when I add this. :)

Reply:
Just following examples from SM :).